frontend/package.json

```diff
@@ -35,14 +35,14 @@
     "@uiw/codemirror-theme-atomone": "^4.22.0",
     "@uiw/codemirror-theme-github": "^4.22.0",
     "@vitejs/plugin-react": "^4.0.4",
-    "codemirror-json-schema": "0.7.0",
+    "codemirror-json-schema": "0.7.8",
```

Comment:
we definitely don't want this update as it will make all the syntax files come back. I'll add a PR to ignore this dependency from renovate